src/exp.h

```diff
@@ -33,45 +33,45 @@ inline const RegEx& Tab() {
   return e;
 }
 inline const RegEx& Blank() {
-  static const RegEx e = Space() || Tab();
+  static const RegEx e = Space() | Tab();
```

Comment:
Why switch to overload | instead of ||?

Reply:
Overloading the boolean operators generates -Weffc++ warnings since both operands will be evaluated in those overloads. I also though these operators behaved more like when combining bit patterns than boolean operators.

Reply:
OK.